docs/image.rst

```diff
@@ -0,0 +1,108 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class:: Image(width_or_string=None, height=None, buffer=None)
+
+    If ``width_or_string`` is a string, creates an image from that string.
+
+    If it's an integer, a height has to also be specified, and it creates
+    an image with ``width_or_string`` columns and ``height`` rows. Optionally
+    ``buffer`` can be an array of ``width_or_string``×``height`` integers to
+    initialize the image.
```

Comment:
Again, this is awkward, it would be much easier to explain if there was a ``from_string`` classmethod.